lib/src/system/system.dart

```diff
@@ -1,18 +1,28 @@
 part of oxygen;
 
+/// Systems contain the logic for components.
+///
+/// They can update data stored in the components.
+/// They query on components to get the entities that fit their Query.
+/// And they can iterate through those entities each execution frame.
 abstract class System {
-  List<Entity> get entities => world.entities;
-
+  /// The world to which this system belongs to.
   World world;
 
+  /// The priority of this system.
+  ///
+  /// Used to set the priority of this system compared to the other systems.
   int priority = 0;
```

Comment:
can this be changed at runtime? first small or first big?

Reply:
Updated the docs to make this more clear